src/applications/gi/components/profile/SchoolLocations.jsx

```diff
@@ -316,7 +310,7 @@ export default function SchoolLocations({
           </span>
         )}
       </span>
-      {renderFacilityTable(main)}
+      <Router>{renderFacilityTable(main)}</Router>
```

Comment:
We were previously creating `<Link>` react components but there was no `<Router>` around them, which meant they would not render as a link. I am putting a router around the entire table, and now links do render, but maybe there is a better way to handle this.